grammars/less.cson

```diff
@@ -183,13 +141,35 @@
     'name': 'meta.attribute-selector.css'
   }
   {
-    'captures':
+    'begin': '^\\s*((@)import\\b)'
+    'beginCaptures':
       '1':
-        'name': 'keyword.control.at-rule.import.css'
+        'name': 'keyword.control.at-rule.import.less'
       '2':
-        'name': 'punctuation.definition.keyword.css'
-    'match': '^\\s*((@)import\\b)'
+        'name': 'punctuation.definition.keyword.less'
+    'end': ';'
+    'endCaptures':
+      '0':
+        'name': 'punctuation.terminator.rule.css'
     'name': 'meta.at-rule.import.css'
```

Comment:
This will target the entire rule as `meta.at-rule.import.css`, including any leading whitespace due to the begin rule.